core-primitives/stf-executor/src/traits.rs

```diff
@@ -43,35 +43,37 @@ pub trait StfExecuteShieldFunds {
 
 /// Execute a trusted call on the STF
 pub trait StfExecuteTrustedCall {
-	fn execute_trusted_call<PB>(
+	fn execute_trusted_call<PH>(
```

Comment:
This is something I'd like to discus with you as well (not specific to this PR): I think we should also use meaningful names for the type parameters, and not abbreviations - same rules as for all variable names. I'm willing to have lengthy definitions for the benefit of readability (especially for now developers, who don't know `PH` = ParentchainHeader).

Reply:
Totally agreed. I don't like it either, but I didn't want to introduce too many new things in this PR. Removed the block only because it was easier to write tests.. didn't want to write a block when it's not even needed anywhere.